mpas_analysis/shared/plot/plotting.py

```diff
@@ -164,15 +164,14 @@ def timeseries_analysis_plot(config, dsvalues, N, title, xlabel, ylabel,
                   'color': config.get('plot', 'titleFontColor'),
                   'weight': config.get('plot', 'titleFontWeight')}
 
+    plot_xtick_format(plt, calendar, minDays, maxDays, maxXTicks)
```

Comment:
I moved this up because it updates the limits of the x axis (so they're "tight').  If we do this first, the call to `get_xlim()` will give us the desired, tighter axis.